packages/rs-drive-abci/src/execution/validation/state_transition/state_transitions/documents_batch/data_triggers/triggers/dpns/v0/mod.rs

```diff
@@ -123,6 +130,21 @@ pub fn create_domain_data_trigger_v0(
             result.add_error(err);
         }
 
+        if normalized_parent_domain_name
+            != convert_to_base58_chars(parent_domain_name.to_lowercase().as_str())
+        {
+            let err = DataTriggerConditionError::new(
+                data_contract.id(),
+                document_transition.base().id(),
+                format!(
+                    "Normalized parent domain name doesn't match parent domain name: {} != {}",
+                    normalized_parent_domain_name, parent_domain_name
+                ),
+            );
+
+            result.add_error(err);
+        }
```

Comment:
Why?

Reply:
`parentDomainName` we use to construct the whole original domain. The normalized one we need for resolving. So here we verify that a user sent correct normalized form.